packages/coding-standard/config/symplify-rules.neon

```diff
@@ -239,6 +239,14 @@ services:
             forbiddenClassMethods:
                 PHPStan\Rules\Rule: ['getRule']
 
+    -
+        class: Symplify\CodingStandard\Rules\ForbidNewOutsideFactoryServiceRule
+        tags: [phpstan.rules.rule]
+        arguments:
+            #types:
+            #    - *Search
```

Comment:
In case of configurable rule, the empty config does not do anything actually. 

But it can be fully commented out there, just as an example

Reply:
I've fully commented it